tests/CMakeLists.txt

```diff
@@ -90,7 +90,7 @@ function(add_unit_test name)
            COMMAND bsdtestharness $<TARGET_FILE:${name}>)
   set_tests_properties(${name}
                        PROPERTIES
-                         TIMEOUT 30
+                         TIMEOUT 600
```

Comment:
right.  30 is not long enough.  Swift CI machines get very loaded.   I'm seeing dispatch_select routinely take 40 or 50 seconds.